src/components/AccountBox/AccountBox.jsx

```diff
@@ -0,0 +1,63 @@
+import './AccountBox.scss';
+
+import React, { PureComponent, PropTypes } from 'react';
+
+import cxHelpers from 'lib/decorators/classNameHelpers';
+import Avatar from 'components/Avatar';
+
+@cxHelpers("AccountBox")
+class AccountBox extends PureComponent {
+  constructor() {
+    super()
+    this.state = {
+      expanded: false
+    }
+  }
+  static propTypes = {
+    username: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+    expanded: PropTypes.bool.isRequired,
+    avatarURI: PropTypes.string
+  };
+
+  handleClick = () => {
+    console.log('click')
+    this.setState({ expanded: !this.state.expanded });
+    // this.props.onClick();
+  };
+
+  render(){
+    const { username, avatarURI } = this.props;
+    const { expanded } = this.state;
+
+    return(
+      <div className={this.cx({ expanded })}>
+        <a className={this.cxEl('header', { expanded })} onClick={this.handleClick}>
+          <Avatar className={this.cxEl('avatar')} uri={avatarURI} />
+
+          <span className={this.cxEl('name')}>{username}</span>
+
+          <div className={this.cxEl('menu-icon', { expanded })}>
+            {['one', 'two', 'three'].map((m, i) =>
```

Comment:
I'm not sure I understand what this is doing.  Does it show which div is clicked on by being highlighted or something?  Whenever I see an array like this of potentially infinite numerations (why does it stop at three), I'm always a little skeptical that it's the right design pattern.

But like I said, not totally sure what this and the line below are trying to do, so it could be the perfect design pattern :)